server/sonar-web/src/main/js/apps/issues/facets/severity-facet.js

```diff
@@ -26,7 +26,7 @@ export default BaseFacet.extend({
 
   sortValues (values) {
     const order = ['BLOCKER', 'MINOR', 'CRITICAL', 'INFO', 'MAJOR'];
-    return _.sortBy(values, function (v) {
+    return _.sortBy(values, v => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)